src/webserver/webserver.py

```diff
@@ -32,7 +32,16 @@ def upload_file():
         if file and allowed_file(file.filename):
             filename = secure_filename(file.filename)
             file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
-            return redirect(url_for('upload_file',
+            # redirects to analysis page
+            return redirect(url_for('analysis',
                 filename=filename))
 
-    return render_template('index.html')
\ No newline at end of file
+    return render_template('index.html')
+
+@app.route('/analysis')
+def analysis():
+    file = os.listdir('uploads\\')[0]
+    print(file)
+    os.remove(f'uploads\\{file}')
+    return render_template('analysis.html', file=file.strip())
```

Comment:
I made this code and I think its bad and needs fixing... but it works and this project is made by me so I can do what I want